specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2024-05-01/appconfiguration.json

```diff
@@ -0,0 +1,2890 @@
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next set of paged results.",
+          "type": "string"
+        }
+      }
+    },
+    "Replica": {
+      "description": "The replica resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the replica.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "The location of the replica.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "Resource system metadata.",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/ReplicaProperties",
+          "description": "All replica properties.",
+          "x-ms-client-flatten": true,
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ReplicaProperties": {
+      "description": "All replica properties.",
+      "type": "object",
+      "properties": {
+        "endpoint": {
+          "description": "The URI of the replica where the replica API will be available.",
+          "type": "string",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the replica.",
+          "enum": [
+            "Creating",
+            "Succeeded",
+            "Deleting",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ReplicaProvisioningState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Snapshot": {
+      "description": "The snapshot resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the snapshot.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/SnapshotProperties",
+          "description": "All snapshot properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-azure-resource": true
```

Comment:
use proxyresource definition from Commontypes

Reply:
We are stabilizing the preview api, this definition is consistent with what we have in hte existing preview version api spec. [link](https://github.com/Azure/azure-rest-api-specs/blob/2e5be0e72597c6fc8d438f20e38087d900c16427/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2023-09-01-preview/appconfiguration.json#L2709C7-L2709C34)